api/api.yaml

```diff
@@ -215,16 +215,27 @@ paths:
           $ref: '../result/request.yaml#/components/responses/InternalServerError'
 
   /results/state:
+    description: >-
+      State API
     post:
-      tags: ['Metrics']
-      operationId: get_state_metrics
+      tags: ['State']
+      operationId: get_states
+      requestBody:
+        description: >-
+          Request to traffic generator for states of choice
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '../result/states.yaml#/components/schemas/States.Request'
       responses:
         '200':
-          description: List of states
+          description: >-
+            Response from traffic generator for chosen states
```

Comment:
"Response from traffic generator" this phrase is not needed. Jut talk about what this response contents.

Reply:
This one is a generic description (also consistent with 'get_metrics' response description). As response contents are not limited to ipv4/v6 neighbor state, instead, it can vary, depending on what choices are added in the future, details of the available choices are not mentioned here.